cfme/fixtures/v2v_fixture.py

```diff
@@ -0,0 +1,503 @@
+import fauxfactory
+import itertools
+import pytest
+import json
+from collections import namedtuple
+from riggerlib import recursive_update
+from widgetastic.utils import partial_match
+
+from cfme.fixtures.provider import setup_or_skip
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+
+
+FormDataVmObj = namedtuple("FormDataVmObj", ["form_data", "vm_list"])
+
+
+@pytest.fixture(scope="session")
+def v2v_downstream_enable(appliance):
+    """downstream 5.10 public beta has migration option disabled
+        Edit config and restart evm to enable it for testing
+        Not needed after GA
+    """
+    migration_check = appliance.ssh_client.run_command(
+        'grep "migration" /var/www/miq/vmdb/config/permissions.yml'
+    )
+    if appliance.version.is_in_series("5.10") and migration_check.rc == 1:
+        # 5.10 and migration is not in permissions.yml
```

Comment:
Q: this is going to continue for upcoming builds like `5.11?` if yes I would suggest to use `appliance.version`check.

Reply:
Removed